CollAction/Models/ProjectViewModels/FindProjectsViewModel.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.Extensions.Localization;
+using System;
+
+namespace CollAction.Models
+{
+    public class FindProjectsViewModel
+    {
+        private readonly IStringLocalizer _localizer;
+
+        public FindProjectsViewModel(IStringLocalizer localizer)
+        {
+            _localizer = localizer;
+        }
+
+        public int ProjectId { get; set; }
+
+        public string ProjectName { get; set; }
+
+        public string ProjectProposal { get; set; }
+
+        public string CategoryName { get; set; }
+
+        public string CategoryColorHex { get; set; }
+
+        public string LocationName { get; set; }
+
+        public string BannerImagePath { get; set; }
+
+        public string BannerImageDescription { get; set; }
+
+        public string DescriptiveImagePath { get; set; }
+
+        public string DescriptiveImageDescription { get; set; }
+
+        public int Target { get; set; }
+
+        public int Participants { get; set; }
+
+        public int ProgressPercent
+        {
+            get
+            {
+                return Participants * 100 / Target;
```

Comment:
Is target default initialized to >0 somewhere so that we won't get exception for division by 0 ?

Reply:
Ok project cannot be created with 0 target I think by validation but could be possible to update it by admin to 0 ?

Reply:
I checked, the admin can't update it to 0 as well. This is already how it works, just moved it around little.